utils/mock-data-loader/tfm/mocks/users.js

```diff
@@ -135,4 +135,12 @@ module.exports = [
     firstName: 'Caroline-Test',
     lastName: 'O’Test Taylor',
   },
+  {
+    username: 'ESTORE',
+    email: 'test@testing.com',
+    teams: ['ESTORE'],
+    timezone: 'Europe/London',
+    firstName: 'ESTORE',
+    lastName: 'ESTORE',
+  },
```

Comment:
Can see this is being used to send errors - we should make a note somewhere to make sure this is a real email address when doing data migration or go-live check list

Reply:
yes, I've updated it so instead of assigning it to DEVs only, it sends it to anyone who is part of ESTORE team